tests/integration/staking/keeper/determinstic_test.go

```diff
@@ -5,23 +5,17 @@ import (
 	"time"
 
 	"cosmossdk.io/math"
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
```

Comment:
Are we still using `require` here? Is this intended?

Reply:
yes, `require` is needed in `DeterministicIterations` function from testdata package.
We can leave this as of now.
I'll be creating a follow-up PR which removes testify usage `testdata` package which will eventually end up cleaning testify in tests/integration

Reply:
As I see, there are very less number (mostly 3 to 4) of usages of require in `testdata` package. Maybe they can be included in this PR. wdyt?

Reply:
I've already created one: https://github.com/cosmos/cosmos-sdk/pull/14575